packages/cosmo-dycore/package.py

```diff
@@ -21,10 +21,14 @@ class CosmoDycore(CMakePackage):
             git='ssh://git@github.com/MeteoSwiss-APN/cosmo.git',
             branch='mch')
     version('5.09a.mch1.2.p2', git=apngit, tag='5.09a.mch1.2.p2')
+    version('dev-build',
```

Comment:
There's not a single package in spack with a version called `dev-build` and I don't think spack-c2sm should have one either, because `spack install cosmo-dycore@dev-build` could easily be confused with `spack dev-build cosmo-dycore @my_build_1` because of the name clash.

Reply:
version `dev-build` was introduced to still be able to install cosmo, even if the upstream instance installed master.
If upstream installed version `master` one could not install it with the local instance, because spack said:
Version `master` already installed. And uninstaling version in the upstream was of course not possible for individual users.

Long story short:
We can get-rid of dev-build version, in case we do not use an upstream that install cosmo/cosmo-dycore

Reply:
Aha